src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -3119,7 +3119,7 @@ A catch() block after a catch (System.Exception e) block can catch non-CLS excep
     <value>A namespace declaration cannot have modifiers or attributes</value>
   </data>
   <data name="ERR_BadPlatformType" xml:space="preserve">
-    <value>Invalid option '{0}' for /platform; must be anycpu, x86, Itanium, arm, arm64 or x64</value>
+    <value>Invalid option '{0}' for /platform; must be anycpu, x86, Itanium, arm, arm64, loongarch64 or x64</value>
```

Comment:
This and other changes related to loongarch64 value of platform switch should be reverted too.